extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,15 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_nativeConfig': {
```

Comment:
@avimehta,

We have doc and example on our site. Would you be okay with a link to those docs and samples?